content/chapters/compute/lab/content/threads.md

```diff
@@ -6,8 +6,8 @@ Compile the code in `support/sum-array/d/sum_array_threads.d` and run it using 1
 Each thread runs the `calculateArrayPartSum` function and then finishes.
 Running times should be _slightly_ smaller than the implementation using processes.
 This slight time difference is caused by process creation actions, which are costlier than thread creation actions.
-Because a process needs a separate virtual address space (VAS) and needs to duplicate some internal structures such as the file descriptor table and page table, it takes the operating system more time to create it than to create a thread.
-On the other hand, threads belonging to the same process share the same VAS and, implicitly, the same OS-internal structures.
+Because a process needs a separate `VAS` (**virtual address space**) and needs to duplicate some internal structures such as the file descriptor table and page table, it takes the operating system more time to create it than to create a thread.
```

Comment:
```suggestion
Because a process needs a separate **VAS (virtual address space)** and needs to duplicate some internal structures such as the file descriptor table and page table, it takes the operating system more time to create it than to create a thread.
```